webapp/components/Table/Content/Content.js

```diff
@@ -44,7 +42,7 @@ const TableContent = (props) => {
               className={className}
               style={{ gridTemplateColumns }}
             >
-              {React.createElement(rowComponent, { ...props, idx: i, offset, row })}
+              {React.createElement(rowComponent, { idx: i, offset, row, rowNo: i + offset + 1, active })}
```

Comment:
should we propagate the rest of props as we did before?

Reply:
nope